app/http.php

```diff
@@ -20,7 +20,7 @@
 
 $http = new Server("0.0.0.0", App::getEnv('PORT', 80));
 
-$payloadSize = max(4000000 /* 4mb */, App::getEnv('_APP_STORAGE_LIMIT', 10000000 /* 10mb */));
+$payloadSize = 5 * (1024 * 1024); // 5MB
```

Comment:
@eldadfux If we set this to 5 MB, Any files above 5MB will have to chunked, this will be the limit for chunking during upload from SDKs and REST API

Reply:
Sounds like the right logic, we can re-think later if we want to increase this limit.

Reply:
Based on S3 multipart upload, we might want to make this at lease 6MB, as S3 explicitly requires each chunk to be at lease 5MB during multipart upload. If we want to support 5MB file chunk , this needs to be above 5 MB as request contains 5MB chunk + other data